src/SprykerEco/Zed/UnzerApi/Business/Api/Response/Converter/UnzerApiResponseConverter.php

```diff
@@ -51,7 +51,7 @@ public function convertUnzerApiGuzzleResponseToUnzerApiResponseTransfer(
 
         $unzerApiResponseTransfer = $this->createUnzerApiResponseTransfer($isSuccessful);
 
-        if (!$isSuccessful) {
+        if (!$isSuccessful || !$unzerApiResponseTransfer->getIsSuccessful()) {
```

Comment:
Why do we need it? On the previous line, the transfer is created whit `isSccessful` field that is set based on `$isSuccessful` variable.